environments/nle.yml

```diff
@@ -7,6 +7,6 @@ A:
 txt:
 
 cname:
-  - name: "hmcts-nle.azurefd.net"
+  - name: "darts-service"
```

Comment:
this will create a record called:

`darts-service.nle.platform.hmcts.net`

pointing to a CNAME of `nle.platform.hmcts.net`

Which doesn't exist.

Is that what you want?

Reply:
Corrected it.

Reply:
Missing afdverify record https://hmcts.github.io/ways-of-working/path-to-live/public-dns.html#setup-required-for-an-application